docs/private_gems.md

```diff
@@ -138,3 +138,35 @@ Gemstash you are interacting with private gems. Gemstash will only let you
 unyank from `/private`. Unlike pushing, Rubygems doesn't support `--host` for
 unyank and yank (yet), so you need to specify the host via the `RUBYGEMS_HOST`
 environment variable.
+
+## Protected Fetching
+
+Private gems often require protected fetching. While the feature is still being discussed at here: https://github.com/bundler/gemstash/issues/24, a temporary solution is available through configuring web server.
+
+Depends on your choice of the web server, for example, Nginx has a `basic_auth` module, which helps to setup HTTP Basic Authentication. On the Bundler side, HTTP Basic Auth credentials can be configured through: http://bundler.io/man/gemfile.5.html#CREDENTIALS-credentials
+
```

Comment:
@pcarranza Sure thing. Please let me know if this helps: https://github.com/taoza/gemstash/commit/c599aab86034507e7eb91c51ab4e515a790cd657